build/BundledDotnetTools.props

```diff
@@ -0,0 +1,6 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <BundledDotnetTools Include="dotnet-watch" Version="$(MicrosoftAspNetCoreAllPackageVersion)" />
```

Comment:
This name of the asp.net package version property has changed. Maybe you should use the AspNetCoreVersion property, instead of the package property?

Reply:
Changed to AspNetCoreVersion